python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -124,7 +124,9 @@ def execute(
 
         if not isinstance(dag, InputDataBuffer):
             # Note: DAG must be initialized in order to query num_outputs_total.
-            self._global_info = ProgressBar("Running", dag.num_outputs_total())
+            self._global_info = ProgressBar(
+                "Running", dag.num_outputs_total(), unit="blocks"
```

Comment:
@bveeramani do you think we should use `bundle` as the unit on the progress bar, or is `block` more user-friendly? One issue is that a`RefBundle` can have multiple `Block`s in the case of block splitting, so `block` would not be entirely accurate in some cases.

Reply:
IMO let's use bundle since block isn't accurate. That said, agree that it's not great to expose bundle as a concept. 

We expose blocks to the user more than we expose bundles, but ideally I don't think we should expose either. Wdyt about replacing the unit with rows or batches in a follow-up PR?

Reply:
As an aside, we mix blocks and bundles in the code. For example, `_estimated_output_blocks` should really be `_estimated_output_bundles`. I'll open a PR to clean this up:
https://github.com/bveeramani/ray/blob/ab5b7c3b238a688802dcc87399cc3b6afbbc5603/python/ray/data/_internal/execution/operators/map_operator.py#L325-L330

Reply:
PR here https://github.com/ray-project/ray/pull/46547